sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -4674,23 +4674,34 @@ case class ArrayExcept(left: Expression, right: Expression) extends ArrayBinaryL
 @ExpressionDescription(
   usage = """
     _FUNC_(x, pos, val) - Places val into index pos of array x.
-      Array indices start at 1, or start from the end if index is negative.
+      Array indices start at 1. The maximum negative index is -1 for which the function inserts
```

Comment:
What is the meaning of `maximum negative index` here? Is it mathematically to indicate `-1` is the biggest among all negative values for `pos` parameter?

Reply:
>  Is it mathematically to indicate -1 is the biggest among all negative values for pos parameter?

Correct.

Reply:
Thanks!